library/spark/spark-accumulo-library/src/test/java/uk/gov/gchq/gaffer/sparkaccumulo/operation/handler/scalardd/GetRDDOfElementsHandlerTest.java

```diff
@@ -377,4 +377,22 @@ public void checkGetCorrectElementsInRDDForEdgeId() throws OperationException, I
         sparkContext.stop();
     }
 
+    @Test(expected = java.lang.IllegalArgumentException.class)
+    public void testNoSparkContext() throws OperationException {
+        final Graph graph1 = new Graph.Builder()
+                .addSchema(getClass().getResourceAsStream("/schema/dataSchema.json"))
```

Comment:
instead of this you can do:

```java
final Graph graph = new Graph.Builder()
                .addSchemas(StreamUtil.schemas(getClass()))
                .storeProperties(StreamUtil.storeProps(getClass()))
                .build();
```

This will load all schemas in from resource folder /schema   and load the store properties from a resource file /store.properties.